test/acceptance/file-utils.spec.js

```diff
@@ -36,46 +36,68 @@ describe('lookupFiles', function () {
     }
   });
 
-  (symlinkSupported ? it : it.skip)('should not choke on symlinks', function () {
-    expect(utils.lookupFiles(tmpDir, ['js'], false))
-      .to
-      .contain(tmpFile('mocha-utils-link.js'))
-      .and
-      .contain(tmpFile('mocha-utils.js'))
-      .and
-      .have
-      .length(2);
-    expect(existsSync(tmpFile('mocha-utils-link.js')))
-      .to
-      .be(true);
-    fs.renameSync(tmpFile('mocha-utils.js'), tmpFile('bob'));
-    expect(existsSync(tmpFile('mocha-utils-link.js')))
-      .to
-      .be(false);
-    expect(utils.lookupFiles(tmpDir, ['js'], false))
-      .to
-      .eql([]);
+  describe('.lookupFiles', function () {
+    (symlinkSupported ? it : it.skip)('lookupFiles keke should not choke on symlinks', function () {
+      expect(utils.lookupFiles(tmpDir, ['js'], false))
+        .to
+        .contain(tmpFile('mocha-utils-link.js'))
+        .and
+        .contain(tmpFile('mocha-utils.js'))
+        .and
+        .have
+        .length(2);
+      expect(existsSync(tmpFile('mocha-utils-link.js')))
+        .to
+        .be(true);
+      fs.renameSync(tmpFile('mocha-utils.js'), tmpFile('bob'));
+      expect(existsSync(tmpFile('mocha-utils-link.js')))
+        .to
+        .be(false);
+      expect(utils.lookupFiles(tmpDir, ['js'], false))
+        .to
+        .eql([]);
+    });
+
+    it('should accept a glob "path" value', function () {
+      var res = utils.lookupFiles(tmpFile('mocha-utils*'), ['js'], false)
+        .map(path.normalize.bind(path));
+
+      var expectedLength = 0;
+      var ex = expect(res)
+        .to
+        .contain(tmpFile('mocha-utils.js'));
+      expectedLength++;
+
+      if (symlinkSupported) {
+        ex = ex.and
+          .contain(tmpFile('mocha-utils-link.js'));
+        expectedLength++;
+      }
+
+      ex.and
+        .have
+        .length(expectedLength);
+    });
   });
 
-  it('should accept a glob "path" value', function () {
-    var res = utils.lookupFiles(tmpFile('mocha-utils*'), ['js'], false)
-      .map(path.normalize.bind(path));
+  describe('.files', function () {
+    (symlinkSupported ? it : it.skip)('should not choke on broken symlinks', function () {
```

Comment:
One of the test descriptions specifies "broken" and the other does not; these should be consistent, since it's substantially the same test for the two different functions.